pkgs/misc/drivers/logiops/0_3_3.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  glib,
+  udev,
+  libevdev,
+  libconfig,
+}:
+
+stdenv.mkDerivation (
+  oldAttrs: {
+    pname = "logiops";
+    version = "0.3.3";
+
+    src = fetchFromGitHub {
+      owner = "PixlOne";
+      repo = "logiops";
+      rev = "v${oldAttrs.version}";
+      sha256 = "sha256-9nFTud5szQN8jpG0e/Bkp+I9ELldfo66SdfVCUTuekg=";
+      # In v0.3.0, the `ipcgull` submodule was added as a dependency
+      # https://github.com/PixlOne/logiops/releases/tag/v0.3.0
+      fetchSubmodules = true;
+    };
+
+    nativeBuildInputs = [
+      cmake
+      pkg-config
+    ];
+    buildInputs = [
+      udev
+      libevdev
+      libconfig
+      glib
+    ];
+
+    cmakeFlags = [
+      "-DLOGIOPS_VERSION=${oldAttrs.version}"
+      "-DDBUS_SYSTEM_POLICY_INSTALL_DIR=${placeholder "out"}/share/dbus-1/system.d"
+      "-DCMAKE_BUILD_TYPE=Release"
+    ];
+
+    patches = [ ./pkgs0001-Make-DBUS_SYSTEM_POLICY_INSTALL_DIR-externally-overr.patch ];
+    # Prevent CMake from installing the systemd service by default
+    # https://github.com/PixlOne/logiops/issues/307
+    PKG_CONFIG_SYSTEMD_SYSTEMDSYSTEMUNITDIR = "${placeholder "out"}/lib/systemd/system";
```

Comment:
This comment is confusing: we do "install" the systemd unit to the package. It errors without this as it cannot write to paths outside the (typically `/build`) sandbox.

I don't think this needs much of a comment, except maybe "Specify systemd unit install dir".